build-langs

```diff
@@ -36,13 +36,14 @@ for %langs{ @langs || * }:p.sort: *.key.fc -> (:key($name), :value(%lang)) {
     $proc.sink;
 
     with %lang<size> = $proc.out.slurp(:close).chomp {
-        s/(.)B$/ {$0.uc}iB/;          # "XB" → " XiB"
+        s/(.)B$/ {$0.uc}iB/;          # "XB" → "XiB"
```

Comment:
The space is correct, it intentionally adds a space before the leading character.